Patch rdkafka

omnibs opened the pull request.

stoeffel reviewed (commented):
LGTM!
I wonder if we should just remove the prefix `get`.

jwoudenberg-nri commented:
Yeah, that's a fair point, sounds nicer!

brian-carroll committed:
Modify pause-resume bug example to be more like the kafka fire PHX-1253

brian-carroll committed:
Print a message on polling

brian-carroll committed:
Set default values for fire env vars

omnibs committed:
failed attempt at patching rdkafka

omnibs committed:
successful attempt at patching rdkafka

omnibs committed:
Point straight to the local dir

I was relying on NIV_OVERRIDE_rdkafka=../librdkafka before, but it's easy to forget to do that
